src/encoded/search.py

```diff
@@ -230,29 +231,28 @@ def get_date_range(request):
     return before, after
 
 
-def get_all_results(search):
```

Comment:
Split func into two, ensure returns es_results['hits']['hits'] as generator instead of list if size requested > increment.